packages/bpk-docs/src/template.js

```diff
@@ -31,6 +34,12 @@ export default ({ head = {}, html = '', assets = {} }) => `<!doctype html>
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="robots" content="noindex">
+  <meta property="og:url" content="${SITE_URL}">
+  <meta property="og:type" content="website">
+  <meta property="og:title" content="Backpack">
+  <meta property="og:image" content="${SITE_URL}/${ogImage}">
+  <meta property="og:description" content="Skyscanner's design system.">
```

Comment:
It don't really like this idea of serving the same content for all urls. If we are going to do this shouldn't we serve content specific to the URL?

Reply:
We already use [`react-helmet`](https://github.com/nfl/react-helmet) to manage the document head.

Reply:
@k0nserv I hadn't considered that with the current solution (or lack thereof) it inherits from the page, e.g.:

<img width="214" alt="screen shot 2018-01-16 at 13 23 52" src="https://user-images.githubusercontent.com/73652/34991172-8491fb26-fac0-11e7-8e91-d23c61d39d2c.png">

So I'm going to close this PR and do something smarter with `react-helmet` (thanks @matthewdavidson!) at some point soon.